client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-hosting-details-list.tsx

```diff
@@ -25,9 +28,34 @@ export const useUpgradePlanHostingDetailsList = () => {
 			),
 			icon: trendingUp,
 		};
-		hostingDetails.splice( 1, 1 );
-		hostingDetails.unshift( updatedHostingSpeedDetails );
+		newDefaultHostingDetails = newDefaultHostingDetails.filter(
+			( { id } ) => id !== 'increased-speed'
+		);
+		newHostingDetails.push( updatedHostingSpeedDetails );
+	}
+
+	// FID hosting details.
+	if ( fidPercentageDifference > 0 ) {
+		const wordpressFID = Math.round( 100 * wordpressCwvtechReportJson?.goodFID );
```

Comment:
Following the same approach from LCP, it's using `Math.round`, which means that `99.9%` will be presented as `100%`.
Do you think it's a good approach? Or maybe we should use `Math.floor` in these cases?

Reply:
I'd choose to be consistent with the other metrics.

Reply:
Ah, yeah, makes sense, so long as we're consistent for all the metrics. 👍🏻